@navikt/core/react/src/overlays/action-menu/ActionMenu.stories.tsx

```diff
@@ -0,0 +1,573 @@
+              Radio disabled
+            </ActionMenu.RadioItem>
+            <ActionMenu.RadioItem value="2">Radio</ActionMenu.RadioItem>
+          </ActionMenu.RadioGroup>
+        </ActionMenu.Content>
+      </ActionMenu>
+    );
+  },
+  decorators: [DemoDecorator],
+};
+
+export const TriggerWithTooltip: Story = {
+  render: () => {
+    return (
+      <ActionMenu>
+        <Tooltip content="Tooltip!">
+          <ActionMenu.Trigger>
+            <button>Open action</button>
+          </ActionMenu.Trigger>
+        </Tooltip>
+        <ActionMenu.Content>
+          <ActionMenu.Item onSelect={() => console.log("Item 1 clicked")}>
+            Item 1
+          </ActionMenu.Item>
+          <ActionMenu.Item onSelect={() => console.log("Item 2 clicked")}>
+            Item 2
+          </ActionMenu.Item>
+          <ActionMenu.Item onSelect={() => console.log("Item 3 clicked")}>
+            Item 3
+          </ActionMenu.Item>
+        </ActionMenu.Content>
+      </ActionMenu>
+    );
+  },
+  decorators: [DemoDecorator],
+};
+
+/**
+ * TODO: Bugs
+ * - When keydown "space" on open modal button, the modal is closed instantly.
+ * Unsure if this is because of the keydown-event repeats or if its caused by eventbubbling
+ */
+export const ModalTrigger: Story = {
+  render: () => {
+    const ref = useRef<HTMLDialogElement>(null);
+
+    return (
+      <div>
+        <ActionMenu>
+          <ActionMenu.Trigger>
+            <button>Open action</button>
+          </ActionMenu.Trigger>
+
+          <ActionMenu.Content>
+            <ActionMenu.Item onSelect={() => ref.current?.showModal()}>
+              open modal
+            </ActionMenu.Item>
+            <ActionMenu.Item onSelect={() => console.log("Item 2 clicked")}>
+              Item 2
+            </ActionMenu.Item>
+          </ActionMenu.Content>
+        </ActionMenu>
+        <Modal ref={ref} header={{ heading: "Heading" }}>
+          <Modal.Body>
+            Culpa aliquip ut cupidatat laborum minim quis ex in aliqua.
+          </Modal.Body>
+          <Modal.Footer>
+            <Button type="button" onClick={() => ref.current?.close()}>
+              Close
+            </Button>
+          </Modal.Footer>
+        </Modal>
+      </div>
+    );
+  },
+  decorators: [DemoDecorator],
+};
+
+export const Links: Story = {
+  render: (props) => {
+    return (
+      <ActionMenu open={props.open}>
+        <ActionMenu.Trigger>
+          <button>Open action</button>
+        </ActionMenu.Trigger>
+        <ActionMenu.Content>
+          <ActionMenu.Item as="a" href="#1">
+            Item 1
+          </ActionMenu.Item>
+          <ActionMenu.Item as="a" href="#2">
+            Item 2
+          </ActionMenu.Item>
+          <ActionMenu.Item as="a" href="#3">
+            Item 3
+          </ActionMenu.Item>
+        </ActionMenu.Content>
+      </ActionMenu>
+    );
+  },
+  decorators: [DemoDecorator],
+};
+
+export const Chromatic: Story = {
```

Comment:
Some of the menus in the Chromatic-story are overlapping in the [snapshot](https://www.chromatic.com/test?appId=66b4b3beb91603ed0ab5c45e&id=6706ca4efa67a3c0b8bcc44c).